### PR TITLE
QML: Enable multisampling

### PR DESCRIPTION
**Pull request by Holzhaus**
This makes the spinnies and knob arcs from #3907 look nicer.

### CODE REVIEW

**Comment by Swiftb0y** [comment]
Would it make sense to put this behind a config option so people on weaker systems can benefit from the lower resource consumption?

File: src/skin/qml/qmlskin.cpp [hunk 1]
```diff
@@ -109,6 +109,12 @@ QWidget* QmlSkin::loadSkin(QWidget* pParent,
     qmlRegisterType<QmlControlProxy>("Mixxx", 0, 1, "ControlProxy");
 
     QQuickWidget* pWidget = new QQuickWidget(pParent);
+
+    // Enable multisampling for much nicer rendering of shapes/images
+    QSurfaceFormat format;
+    format.setSamples(2);
```
Comment (uklotzde) [inline src/skin/qml/qmlskin.cpp:115]: ...and we should define a constant for easy discoverability of this customization.

**Comment by uklotzde** [comment]
> Would it make sense to put this behind a config option so people on weaker systems can benefit from the lower resource consumption with multisampling disabled?

Maybe it is sufficient to note this as a TODO together with the constant. Let's implement the default case first and then decide later which degree of customization is actually required.

**Comment by Holzhaus** [comment]
Done.

**Review by uklotzde — APPROVED** [review]
LGTM

**Pull request merged**